### PR TITLE
renewed landing page

### LINKED ISSUE #37: Add a Footer for the application
### Type
Enhancement

### Description
Add a basic footer with copyright symbol. Also you can add a text like `Made with ❤️ for the people of India.`

### Tools
HTML
CSS

### PR DESCRIPTION
**Pull request by Biswa-bob**
## Related Issue
Completely made the landing page.
closes #37 , #38 & #40 

## Proposed Changes

-adding of scroll button 
-adding of new pages

## Check List

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if appropriate):

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by sameersrivastava13** [comment]
Impressive @Biswa-bob. You did a great job. Just one issue I have with the images quality that are getting stretched too much.
Rest everything looks good. What are your thoughts @Sejal2801?. Also you have made the entire page responsive which is good too.   

@Sejal2801 I guess he did a good job should we merge it or you have some other suggestions?

File: index.html [hunk 1]
```diff
@@ -1,89 +1,294 @@
...
+            <h1>Vaccine Tracker</h1>
+            <p>Our goal is to help the world free from the ongoing pandemic.</p>
+          </div>
+          <div class="footer-col-3">
+            <h3>Useful Links</h3>
+            <ul>
+              <li>Coupons</li>
+              <li>Blog Post</li>
+              <li>Return Policy</li>
+              <li>Join Affiliate</li>
```
Comment (sameersrivastava13) [inline index.html:262]: Also @Biswa-bob few extra part you have added in the footer which was not needed. For now, we just wanted the copyright symbol with text `Made with ❤️ for the people of India.`

Comment (Biswa-bob) [reply index.html:262]: sure brother

**Comment by sameersrivastava13** [comment]
@Biswa-bob Also you haven't added the scroll to top button I guess.

**Pull request merged**